R/metadata.R

```diff
@@ -0,0 +1,43 @@
+#' Get plotting metadata for a particualr country
+#'
+#' @param country Country to get metadata for
+#'
+#' @return List of plotting metadata about how to locate data for a specific
+#' indicator, data type and plot type. Also returns metadata about colour
+#' scheme to use for that country and indicator.
+#'
+#' @export
+#'
+#' @examples
+#' get_plotting_metadata("Malawi")
+get_plotting_metadata <- function(country) {
+  metadata <- get_metadata()
+  colour_scale <- get_colour_scale(country)
+  merge(x = metadata, y = colour_scale, by = "indicator")
+}
+
+#' Get colour scale and ranges for a particular country
+#'
+#' @param country Country to get scale for
+#'
+#' @return List of scale information including colour as a d3 scale chromatic
+#' function name, whether to invert the scale and a min and max value for the
+#' scale.
+#'
+#' @keywords internal
+get_colour_scale <- function(country) {
+  scales <- naomi_read_csv(system.file("extdata", "meta", "colour_scales.csv",
```

Comment:
I would recommend at least adding a `mustWork = TRUE` here (`system.file` has a terrible default) but perhaps also adding a wrapper function for this if you think you'll go past the 2 that are here

Reply:
I always forget about this, I've fixed that now.